{"name": "Divide the Two number -- @1.3 ", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'first_number'\n", "trace": "  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"..\\steps\\steps.py\", line 50, in step_impl\n    if int(context.first_number) == int(context.second_number):\n  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "Given take the \"-2\" and \"0\"", "status": "passed", "start": 1685414063775, "stop": 1685414063775}, {"name": "When verify if greater number between \"-2\" and \"0\"", "status": "passed", "start": 1685414063776, "stop": 1685414063776}, {"name": "And divide greater with lower", "status": "passed", "start": 1685414063777, "stop": 1685414063777}, {"name": "Then Print the total result", "status": "passed", "start": 1685414063777, "stop": 1685414063778}, {"name": "And if both are same, return 0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'first_number'\n", "trace": "  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"..\\steps\\steps.py\", line 50, in step_impl\n    if int(context.first_number) == int(context.second_number):\n  File \"C:\\Users\\Jatin-PC\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "start": 1685414063778, "stop": 1685414063780}], "parameters": [{"name": "first_number", "value": "-2"}, {"name": "second_number", "value": "0"}], "start": 1685414063773, "stop": 1685414063781, "uuid": "7b531540-446c-4f9e-816b-4432bedb767c", "historyId": "f1155c89f240aa6454ddddea47df8a50", "fullName": "Perform an division Feature: Divide the Two number", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Perform an division Feature"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}